product-service/src/functions/createProduct/handler.ts

```diff
@@ -0,0 +1,45 @@
+import { dbOptions } from '@functions/dboptions';
+import { Product } from '@functions/types';
+import { AppError } from '@libs/appError';
+import { middyfy } from '@libs/lambda';
+import {
+  ReasonPhrases,
+  StatusCodes
+} from 'http-status-codes';
+import { Client } from 'pg';
+import 'source-map-support/register';
+import { v4 as uuidv4 } from 'uuid';
+
+const createProduct = async ({ body }: { body: Product }): Promise<string> => {
+  const client = new Client(dbOptions);
+  try {
+    await client.connect();
+
+    const { id: productId, title, description, price, count } = body;
```

Comment:
It is not a good idea to insert id in request. Id should be generated in db, not in your be or by user.